packages/devtools/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "devtools",
+  "version": "5.7.5",
+  "description": "A WebDriver extension that runs commands on the DevTools protocol",
```

Comment:
Is it really a WebDriver extension? I think puppeteer has nothing to do with it, no?